fiftyone/utils/huggingface.py

```diff
@@ -63,6 +64,26 @@
 
 logger = logging.getLogger(__name__)
 
+def list_hub_datasets():
+    """Lists all FiftyOne datasets available on the Hugging Face Hub.
+
+    This function prints instructions for loading datasets and displays a list of
+    all datasets published by Voxel51 on the Hugging Face Hub.
+    """
+    from huggingface_hub import list_datasets
+    
+    print('You can load any of the following datasets as follows:\n')
+    print('from fiftyone.utils.huggingface import load_from_hub')
+    print('dataset = load_from_hub("Voxel51/dataset-name")\n')
+    print('Datasets on the Hugging Face Hub:\n')
+    
+    # List all datasets for Voxel51
+    org_datasets = list_datasets(author="Voxel51")
+    
+    for dataset in org_datasets:
+        dataset_name = dataset.id.split('Voxel51/')[-1]
```

Comment:
I would argue that `Voxel51/` prefix should be kept. Then the example code will be much cleaner:

```py
import fiftyone.utils.huggingface as hf

# List all FiftyOne-compatible datasets in the HF Hub
names = hf.list_hub_datasets()
print(names)

# Load a dataset
dataset = hf.load_from_hub(names[0])
```